ui/package-lock.json

```diff
@@ -33,7 +33,7 @@
         "typescript": "5.3.3"
       },
       "engines": {
-        "node": "18"
+        "node": "20"
```

Comment:
Might wanna drop this change

Reply:
I'll drop it then, this was just to align with package.json